third_party/ienn/src/ie_compilation.cpp

```diff
@@ -588,8 +703,12 @@ int32_t Compilation::AddFullyConnectedV1(const Operation& operation) {
   try {
     const uint32_t weights_index = operation.inputs[1];
     const uint32_t bias_index = operation.inputs[2];
-    AddConstant(weights_index, true);
-    AddConstant(bias_index, true);
+    if (index_op_map_.find(weights_index) == index_op_map_.end()) {
```

Comment:
Line 244 has been added, so pls delete it here.

Reply:
done.